packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -139,6 +139,40 @@ class SkiaGoldClient {
     }
   }
 
+  /// Prepares the local work space for golden file testing and calls the
+  /// goldctl `auth` command using the luci environment.
+  ///
+  /// This ensures that the goldctl tool is authorized and ready for testing. It
+  /// will only be called once for each instance of
+  /// [FlutterSkiaGoldFileComparator].
+  Future<void> luciAuth() async {
```

Comment:
Would it make sense to detect if it's a LUCI environment in `auth()` and then do this work? @kjlubick Is there a way to detect a LUCI host?

Reply:
🤦‍♀ Much better idea. Thank you! Detecting luci is easy, the environment has a `SWARMING_TASK_ID` variable. I'll update this tomorrow.